ports/esp32/modules/machine.py

```diff
@@ -0,0 +1,78 @@
+from umachine import *
+
+import esp32
+
+if hasattr(esp32, "PCNT"):
+    class Counter:
+        _PCNT = esp32.PCNT
+        RISING = 1
+        FALLING = 2
+        UP = _PCNT.INCREMENT
+        DOWN = _PCNT.DECREMENT
+
+        def __init__(self, unit_id, *args, **kwargs):
+            self._pcnt = self._PCNT(unit_id, minimum=-32000, maximum=32000)
+            self._pcnt.irq(self._wrap, self._PCNT.IRQ_MINIMUM | self._PCNT.IRQ_MAXIMUM)
+            self.init(*args, **kwargs)
```

Comment:
These two lines need to be swapped. Calling init re-initializes the counter and thus disables the IRQs again.

Reply:
That's what I get for making code changes when I should have just been rebasing…